### PR TITLE
universal-query: API testing with multi named vectors

### PR DESCRIPTION
**Pull request by agourlay**
This PR introduces test for the new query API using several named vectors of different types:
- dense
- sparse
- multivector

The goal is to provide a larger safety net as we start to change the internals significantly.

### CODE REVIEW

File: tests/openapi/openapi_integration/test_query_full.py [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+        method="POST",
+        path_params={"collection_name": collection_name},
+        body={
+            "order_by": "count",
+        },
+    )
+    assert response.ok
+    scroll_result = response.json()["result"]["points"]
+
+    query_result = root_and_rescored_query({"order_by": "count"}, "dense-image", with_payload=True)
```
Comment (coszio) [inline tests/openapi/openapi_integration/test_query_full.py:348]: `"dense-image"`?

File: tests/openapi/openapi_integration/test_query_full.py [hunk 2]
```diff
@@ -0,0 +1,551 @@
...
+    search_result_3 = response.json()["result"]
+
+    response = request_with_validation(
+        api="/collections/{collection_name}/points/search",
+        method="POST",
+        path_params={"collection_name": collection_name},
+        body={
+            "vector": {
+                "name": "dense-multi",
+                "vector": [3.05, 3.61, 3.76, 3.74], # legacy API expands single vector to multiple vectors
```
Comment (coszio) [inline tests/openapi/openapi_integration/test_query_full.py:424]: nice check 👍

File: tests/openapi/openapi_integration/test_query_full.py [hunk 3]
```diff
@@ -0,0 +1,551 @@
...
+            "query": {"fusion": "rrf"}
+        },
+    )
+    assert response.ok, response.json()
+    rrf_result = response.json()["result"]
+
+    def get_id(x):
+        return x["id"]
+
+    # rrf order is not deterministic with same scores, so we need to sort by id
```
Comment (coszio) [inline tests/openapi/openapi_integration/test_query_full.py:472]: Non-related to this PR: We could make it deterministic by using `ScoredPointTies`, but do we want to?

**Review by coszio — APPROVED** [review]
Thanks for the safety net 🥅

File: tests/openapi/openapi_integration/test_query_full.py (hunk 1 above)
Comment (coszio) [reply tests/openapi/openapi_integration/test_query_full.py:348]: I meant specifying "using" for order by

Comment (agourlay) [reply tests/openapi/openapi_integration/test_query_full.py:348]: Good catch, forgot that one :face_with_head_bandage: 
Will fix with the next batch of tests :)

**Pull request merged**